pkgs/tools/typesetting/sile/default.nix

```diff
@@ -40,15 +41,31 @@ stdenv.mkDerivation rec {
   LUA_PATH = luaPath;
   LUA_CPATH = luaCPath;
 
+  FONTCONFIG_FILE = makeFontsConf {
+    fontDirectories = [
+      gentium
+      gentium-book-basic
+      dejavu_fonts
+    ];
+  };
+
+  buildPhase = ''
+   make all documentation/developers.pdf documentation/sile.pdf
```

Comment:
Sorry I forgot that comment: What about putting it in `postBuild`? That would leave the nix-magic of the buildPhase intact.
```
postBuild = ''
  make documentation/developers.pdf documentation/sile.pdf
'';
```